Gemfile

```diff
@@ -6,9 +6,8 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gemspec
 
-gem 'activesupport', '4.2.11.3'
```

Comment:
removed this as cocoapods now provides `> 5`

I figured we dont explicitly care for it anyway.